user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_TakeConfigurationBackup.md

```diff
@@ -0,0 +1,200 @@
+- [Step 5: Retrieve the device configuration](#step-5-retrieve-the-device-configuration)
+
+## Step 1: Create the CI Type
+
+First, a [CI Type](xref:CI_Types) needs to be created for IDP to know how to manage the element or device.
+
+Instead of creating the CI Type from scratch, you will generate it based on an existing connector. In this example, the [Generic Switch](https://catalog.dataminer.services/details/connector/931) connector is used, but you can use any other connector.
+
+1. Deploy the connector to start from:
+
+   1. Deploy the latest version of the [Generic Switch](https://catalog.dataminer.services/details/connector/931) connector from the DataMiner Catalog.
+
+      > [!TIP]
+      > See also: [Deploying a Catalog item to your system](xref:Deploying_a_catalog_item)
+
+   1. Make sure the latest version is [set as production](xref:Promoting_a_protocol_version_to_production_version).
+
+      ![Generic Switch connector](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_ProtocolTemplates.png)
+
+1. Generate the CI Type:
+
+   1. Open the IDP app.
+
+   1. Go to *Admin* > *CI Types*.
+
+   1. Above the table on the right, click *Generate* to start the wizard.
+
+      ![Start to generate the CI Type script](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_0.png)
+
+   1. In the wizard, select the connector and click *Generate*.
+
+      ![Select the connector](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_1.png)
+
+   1. Click *Finish* to close the wizard.
+
+   1. Switch off the *Show All* button in the top-right corner to locate the generated CI Type more easily.
+
+      ![Generate the CI Type](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_2.png)
+
+## Step 2: Create an element and manage it
+
+1. [Create an element](xref:Adding_elements) that uses the connector you deployed in the previous step.
+
+   You can enter any IP address in the *IP address/host* field, for example `127.0.0.1`.
+
+1. In the IDP app, go to *Inventory* > *Unmanaged*.
+
+1. Click *Refresh* to refresh the table.
+
+   > [!NOTE]
+   > Make sure *Detected CI Type* contains the generated CI Type. If it does not, confirm that the element was created with the production version.
+
+1. Select the recently created element, and click *Manage*.
+
+   ![Unmanaged element](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_UnmanageElement.png)
+
+1. Go to *Inventory* > *Managed* to confirm the element is now managed by IDP.
+
+   ![Managed element](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_ManageElement.png)
+
+## Step 3: Set up the configuration archive
+
+For IDP to store the configurations, it needs to access a local folder or a shared folder.
+
+1. In the *DataMiner IDP* app, go to *Admin* > *Configuration*.
+
+1. Under *DataMiner Configuration Archive*, specify the settings to access the local or shared folder.
```

Comment:
Which username and password should users use for this? Should they create a dedicated user account for this (because that's the impression I get from the screenshot)?

Reply:
They don't have to create a user, as I did, they can use any user with access to the shared folder.